usefulUtilities/botPlayer/assignmentClientPlayer.js

```diff
@@ -0,0 +1,266 @@
+    // END CONSTS_AND_VARS
+    // *************************************
+
+    // *************************************
+    // START PLAYER
+    // *************************************
+    // #region PLAYER
+    
+    
+    // Player Class for the hfr recordings
+    function Player() {
+        this.isPlayingRecording = false;
+        this.recordingFilename = "";
+    }
+
+    
+    // Play the bot
+    function play(fileToPlay) {
+        console.log("play playing " + JSON.stringify(fileToPlay));
+        this.recordingFilename = fileToPlay;
+        var _this = this;
+        
+        Recording.loadRecording(fileToPlay, function(success, url) {
+            if (success) {
+                _this.isPlayingRecording = true;
+
+                Users.disableIgnoreRadius();
+
+                Agent.isAvatar = true;
+
+                Recording.setPlayFromCurrentLocation(false);
+                Recording.setPlayerUseDisplayName(true);
+                Recording.setPlayerUseHeadModel(false);
+                Recording.setPlayerUseAttachments(true);
+                Recording.setPlayerLoop(false);
+                Recording.setPlayerUseSkeletonModel(true);
+                Recording.setPlayerTime(0.0);
+                Recording.setPlayerVolume(1.0);
+                
+                Recording.startPlaying();
+            } else {
+                console.log("Could not load recording " + fileToPlay);
+
+                _this.isPlayingRecording = false;
+                _this.recordingFilename = "";
+                // This should remove the avatars however they are coming back in as white spheres at the origin
+                // Agent.isAvatar = false;
+            }
+        });
+    }
+
+
+    // Stop the bot and remove 
+    // #NOTE:  Adding and removing the agent is currently causing issues.  Using a work around for the meantime
+    function stop() {
+        console.log("Stop playing " + this.recordingFilename);
+
+        if (Recording.isPlaying()) {
+            Recording.stopPlaying();
+
+            // This looks like it's a platform bug that this can't be removed
+            // Agent.isAvatar = false;
+        }
+        this.isPlayingRecording = false;
+    }
+
+    
+    // Check if the bot is playing
+    function isPlaying() {
+        return this.isPlayingRecording;
+    }
+
+
+    Player.prototype = {
+        play: play,
+        stop: stop,
+        isPlaying: isPlaying
+    };
+
+    
+    // #endregion
+    // *************************************
+    // END PLAYER
+    // *************************************
+
+    // *************************************
+    // START MESSAGES
+    // *************************************
+    // #region MESSAGES
+    
+
+    // Handle messages fromt he manager
+    var IGNORE_PLAYER_MESSAGES = ["REGISTER_ME", "ARE_YOU_THERE_MANAGER"];
+    var playInterval = null;
+    var INTERVAL_TIME = 20000;
+    var MINIMUM_PLAY_TIME = 7500;
+    var randomScaler = 1.5
```

Comment:
```suggestion
    var randomScaler = 1.5;
```

Reply:
also, what is this?

Reply:
removing all those for now, but it was just to make the interval last a little longer than the timeout before it played again.